kernel/classes/ezcontentobjecttreenodeoperations.php

```diff
@@ -113,6 +113,7 @@ static function move( $nodeID, $newParentNodeID )
                 $nodeIDList = array( $nodeID );
                 eZSearch::removeNodeAssignment( $node->attribute( 'main_node_id' ), $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList );
                 eZSearch::addNodeAssignment( $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList, true );
+                eZSearch::refreshChildren( $node );
```

Comment:
If we follow what was implemented for delayed indexing, the logic should be put in eZContentOperationCollection::registerSearchObject(). If we follow eZSearch::addNodeAssignment, the 4th parameter for that is $isMoved, which then gets propagated to the search engine implementation and eZContentOperationCollection::registerSearchObject().

Because of the risk of timeouts, I think we should simply make a cronjob a requirement, regardless of whether the delayed indexing flag is on or not. As it is now, the delayed indexing flag is a requirement: https://github.com/ezsystems/ezpublish-legacy/blob/master/kernel/content/ezcontentoperationcollection.php#L571.